draft-pignataro-eimpact-icmp.xml

```diff
@@ -122,10 +122,18 @@
         administrative domain:
         <?rfc subcompact="yes" ?>
         <list style="symbols">
-          <t>Power metrics (e.g., time average, min/max)</t>
+          <t>Present and idle power (node level, component level)</t>
+          <t>Throughput (node level, component level): this allows to determine a relationship between power and traffic load</t>
+          <t>Ecolabels and Environmentally Relevant Certifications (EERCs)</t>
+        </list>
+        <?rfc subcompact="no" ?>
+      </t>
+      <t>
+        Future versions of this document may also specify:
```

Comment:
Not only "of this document", but frankly other documents can add in the future. I would do like
"Additional environmental impact metrics, such as the following, may also be specified in the future:" Or similar.
Although, this introduction says "these exemplary ... metrics", so these are examples.

Reply:
```suggestion
        Additional environmental impact metrics, such as the following, for example, may also be specified in the future:
```

Reply:
Actually, the first list above is no longer a list of examples, but really what's contained in the draft - hence I suggest to take the "exemplary" out above, and that's why I introduced it here: only these are examples.